docs/content/documentation/content/overview.md

```diff
@@ -3,9 +3,7 @@ title = "Overview"
 weight = 10
 +++
 
-Zola uses the directory structure to determine the site structure.
-Each child directory in the `content` directory represents a [section](@/documentation/content/section.md)
-that contains [pages](@/documentation/content/page.md) (your `.md` files).
+Zola determines your site's structure from its directories. Each child folder in the `content` directory represents a separate [section](@/documentation/content/section.md)[^1]that contains [pages](@/documentation/content/page.md) (your `.md` files).
```

Comment:
What do you think of https://zola.discourse.group/t/proposal-deprecate-sections/1968 ?